tests/test_basic_functionality.py

```diff
@@ -458,6 +458,18 @@ def test_to_dict_yaml(test_spec: Path) -> None:
     assert yaml_config == lct.to_dict()
 
 
+def test_getter() -> None:
+    getter_dict: NestedDict = {
+        "outer_layer_1": "test_value",
+        "outer_layer_2": {"inner_layer": "test_value2"},
+    }
+    lct = LayeredConfigTree(getter_dict)
+
+    assert lct.get("outer_layer_1") == "test_value"
+    assert lct.get("outer_layer_2").to_dict() == getter_dict["outer_layer_2"]  # type: ignore [union-attr]
+    assert lct.get("outer_layer_2").get("inner_layer") == "test_value2"  # type: ignore [union-attr]
```

Comment:
Should also add tests to check behaviour for:
1. getting a key that doesn't exist (which would help answer my question above)
2. passing theings into the default_value arg (I still don't really understand what this arg is for)

Reply:
Good call - ugh I've been careless/not thinking straight all day :/

Reply:
@stevebachmeier the point of the default_value arg is just to make this match `dictionary.get()` more closely. Perhaps it's better to leave the default value as `None` and have a user replace it with `{}` if they want to chain calls,

Reply:
@rmudambi Isn't that proposal _precisely_ how `dict.get()` works?